src/modules/getfoods.js

```diff
@@ -0,0 +1,72 @@
+import popUp from './popUp.js';
+import displayCounter from './counterFile.js';
+import { addLikeToItem, getLikesToItems } from './like.js';
+
+const displayData = async (arr) => {
+  const board = document.querySelector('.container-food-cards');
+  board.innerHTML = '';
+  arr.forEach((food) => {
+    const item = document.createElement('article');
+    item.id = food.idMeal;
+    item.classList.add('card-food');
+    item.innerHTML = `
+          <div class='card-title'>
+            <h5>${food.strMeal}</h5>
+          </div>
+          <button type='button' class='btn-recipe'>Recipe</button>
+          <div>
+          </div>
+          <div class='btn-liked'>
+            Like
+          </div>
+          <div class='card-img'>
+           <img src="${food.strMealThumb}" class='img-food'>        
+          </div>     
+      `;
+    /* eslint-disable */
+    const btnLikedElement = item.querySelector('.btn-liked');
+    const printLike = (data) => {
+      const likesReturned = data.find(
+        (element) => element.item_id === food.idMeal
+      );
+      btnLikedElement.innerHTML =
+        likesReturned !== undefined
+          ? `<i class="fas fa-heart"></i> (${likesReturned.likes})`
+          : '<i class="far fa-heart"></i> (0)';
+    };
+    getLikesToItems()
+      .then(printLike)
+      .catch((e) => {
+        return e;
+      });
+
+    btnLikedElement.addEventListener('click', () => {
+      addLikeToItem(food.idMeal);
+      getLikesToItems()
+        .then(printLike)
+        .catch((e) => {
+          return e;
+        });
+        /* eslint-enable */
+    });
```

Comment:
- [ ] Please fix the following behavior:
   - Let's say an item has 0 likes on it. If the user clicks the like button, the counter doesn't update and still shows 0. However, if the button was clicked a second time, the counter would show "1". If then the page is refreshed the likes counter would show the correct amount which would be "2" since the button was clicked twice. TLDR; the likes counter doesn't update with the correct amount of likes if the button was clicked.
   
   - {HINT} This problem is caused by the above code snippet where you call the API to post a `like` every time the button is clicked, and then fetch the likes right after without waiting for the API to update the likes counter. You can solve this issue by awaiting the post action to finish and then fetching the likes which would fetch the latest amount of likes.

Reply:
Thank you for the bug-finding @gemmen29,  It has been taken note of, and changes will be made.

Reply:
This change has been done well...